packages/rosaenlg/src/LanguageFrench.ts

```diff
@@ -140,24 +142,28 @@ export class LanguageFrench extends LanguageImpl {
     return frenchParse(val, { dictHelper: this.dictHelper });
   }
 
-  thirdPossessionTriggerRef(owner: any, owned: any, params: any): void {
+  thirdPossessionTriggerRef(owner: any, owned: any, params: ValueParams): void {
     this.valueManager.value(owned, Object.assign({}, params, { det: 'DEFINITE' }));
     this.spy.appendPugHtml(` de `);
     this.valueManager.value(owner, Object.assign({}, params));
   }
 
-  thirdPossessionRefTriggered(owner: any, owned: any, params: any): void {
+  thirdPossessionRefTriggered(owner: any, owned: any, params: ValueParams): void {
     const det: string = this.getDet('POSSESSIVE', {
       genderOwned: this.genderNumberManager.getRefGender(owned, null),
       genderOwner: null,
       numberOwner: this.genderNumberManager.getRefNumber(owner, params),
       numberOwned: this.genderNumberManager.getRefNumber(owned, params),
+      personOwner: (params && params.personOwner) || 3,

```

Comment:
I would remove || 3
as the default value is already managed later

Reply:
removed